### PR TITLE
Add optional wind direction in currentweather module

### PR DESCRIPTION
**Pull request by roxasvalor**
Optionally add the wind direction (pulled from Openweather API) next to the wind speed in the current weather module. It is turned off by default but by adding the option "showWindDirection" in config.js it can be turned on.

### CODE REVIEW

**Comment by CFenner** [comment]
Remove showWindDirection from default cfg in README. 

Good idea.

**Comment by MichMich** [comment]
Nice one! Before I merge: 
@CFenner: what do you mean with you comment?

**Comment by roxasvalor** [comment]
I had added the showWindDirection variable into the default config at the top of the README. Not sure what I was thinking since none of the other config options show there except location and appid. It is fixed now,

File: modules/default/currentweather/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ modules: [
 		config: {
 			// See 'Configuration options' for more information.
 			location: 'Amsterdam,Netherlands',
-			appid: 'abcde12345abcde12345abcde12345ab' //openweathermap.org API key.
+			appid: 'abcde12345abcde12345abcde12345ab', //openweathermap.org API key.
```
Comment (MichMich) [inline modules/default/currentweather/README.md]: Could you remove the trailing comma?

**Comment by MichMich** [comment]
Just one question (a stupid one): doesn't the cardinal direction depend on your location?

**Comment by roxasvalor** [comment]
Depends on what map you're looking at. The conversion from degrees to the cardinal direction assumes that we're using the north pole in the Arctic as "north" and the south pole in Antarctica as "south". If you use maps with Antarctica on top it would be incorrect though unless openweathermap does any trickery behind the scenes.

**Comment by CFenner** [comment]
Are the openweatherAPI degrees switched on south hemisphere?

**Comment by roxasvalor** [comment]
I don't think so. It appears the degrees provided by the openweatherAPI are Azimuth degrees. Azimuth degrees use True North (North Pole) as North.

So I guess the answer is no. It will not switch your direction but I have sent an email to the openweatherAPI team asking them if this is truly the case.

**Comment by MichMich** [comment]
@roxasvalor could you take a look at https://github.com/MichMich/MagicMirror/blob/v2-beta/modules/default/currentweather/currentweather.js#L313 and https://github.com/MichMich/MagicMirror/blob/v2-beta/modules/default/currentweather/currentweather.js#L315 ? they both say ESE.

**Comment by roxasvalor** [comment]
Thanks, I found it. Check PR #308 for the fix.

**Pull request merged**